packages/isolated-editor/client/components/block-editor-toolbar/more-menu/writing-menu.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuGroup } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import FeatureToggle from '../toggle-feature';
+import OptionToggle from '../toggle-option';
+
+/** @typedef {import('../../../index').BlockEditorSettings} BlockEditorSettings */
+/** @typedef {import('./index').OnClose} OnClose */
+
+/**
+ * Writing menu
+ *
+ * @param {object} props - Component props
+ * @param {OnClose} props.onClose - Close the menu
+ * @param {BlockEditorSettings} props.settings - Settings
+ */
+function WritingMenu( { onClose, settings } ) {
+	const { preview, fullscreen, topToolbar } = settings.iso.moreMenu;
+	const { isFullscreen } = useSelect(
+		( select ) => ( {
+			isFullscreen: select( 'isolated/editor' ).isOptionActive( 'fullscreenMode' ),
+		} ),
+		[]
+	);
+
+	// Anything to show?
+	if ( ! fullscreen && ! preview && ! topToolbar ) {
+		return null;
+	}
+
+	return (
+		<MenuGroup label={ _x( 'View', 'noun' ) }>
+			{ topToolbar && (
+				<>
+					<FeatureToggle
+						feature="fixedToolbar"
+						label={ __( 'Top toolbar' ) }
+						info={ __( 'Access all block and document tools in a single place.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **96** times:
`translate( 'Access all block and document tools in a single place' )` *ES Score: 8*